blockonomics-woocommerce.php

```diff
@@ -496,7 +586,7 @@ function blockonomics_plugin_activation() {
 
     $html = '<div class="updated">';
     $html .= '<p>';
-    $html .= __( 'Congrats, you are now accepting BTC payments! You can configure Blockonomics <a href="options-general.php?page=blockonomics_options">on this page</a>.', 'blockonomics-bitcoin-payments' );
+    $html .= __( 'Congrats, you are now accepting BTC payments! You can configure Blockonomics <a href="page=options-general.php?blockonomics_options">on this page</a>.', 'blockonomics-bitcoin-payments' );
```

Comment:
This URL leads to the incorrect page after plugin activation.

Reply:
It's now been updated.